modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -2,19 +2,73 @@
 
 This kernel adds the friction term to the porous media Navier Stokes momentum
 equations. This kernel must be used with the canonical PINSFV variable set,
-e.g. pressure and superficial velocity.  A variety of models are supported for
-the friction force:
+e.g. pressure and superficial velocity. This kernel supports Darcy and
+Forchheimer friction models:
 
 Darcy drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\epsilon F_i = - f_i \rho \frac{v_{D,i}}{\epsilon}
 \end{equation}
 Forchheimer drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+\epsilon F_i = - f_i \rho \frac{v_{D,i}}{\epsilon}\frac{|v_D|}{\epsilon}
+\end{equation}
+where $F_i$ is the i-th component of the friction force (denoted by $\mathbf{F_f}$ in [!eqref](pinsfv.md#eq:pinsfv_mom)), $f_i$ the friction factor, which may be anisotropic,
```

Comment:
add epsilon in the F_i and F_f to be clearer

Reply:
so you're saying in both `.md` files?

Reply:
no in this one. It s there in the other one

Reply:
In `pinsfv.md` there is a multiplication of `F_f` by `\epsilon`

Reply:
yeah. i thin having eps F_f here eliminates the confusion of whether F is the whole force term or it needs eps to go in the momentum equation

Reply:
This conversation has now confused **me**. I think it would be nice to have matching between the `pinsfv.md` equation and this one, or else I think that could generate confusion in its own right. I would suggest removing the multiplication of the LHS of line 14 by epsilon (because it is nowhere to be seen in the kernel) and then also removing the multiplication by epsilon of the force term in `pinsfv.md`. (I would also probably then explicitly write out epsilon * rho * g for the gravity term)

Reply:
it s not in the kernel because it s based in cL ?

Reply:
Haha well I'm not sure what are "the same equations as everyone". If people generally expect to see an epsilon in the numerator in the kernel (or consequently the removal of an epsilon in the denominator), then I am happy to factor that out of the material calculations as well in this PR

Reply:
I think the epsilon term on F comes from the epsilon factor on rho which is included in the equation above. Maybe I don't get the conversation here.

Reply:
I've now factored epsilon out of the material calculations of Darcy and Forchheimer

Reply:
> Ah okay, I would also be supportive of getting f_i from the material and adding the rho*u(*u) factor in the kernel. 

I believe this is exactly what we are doing now with the exception that for Forchheimer the piece multiplying the Forchheimer "coefficient" is rho*u*speed where u is the superficial velocity and speed is the norm of the interstitial velocity

Reply:
in the pinsfv equation in pinsfv.md, these are multiplied by epsilon. This is not something we have in the kernel

Reply:
This is because the epsilon cancels out with the epsilon in the denominator of `v_D / epsilon`. That's why you don't see it in the kernel. Do you want me to add some more commenting around that?

Reply:
no it s fine. I didnt think of that.